service/frontend/namespace_handler.go

```diff
@@ -585,7 +586,7 @@ func (d *namespaceHandlerImpl) UpdateNamespace(
 		}
 	}
 
-	if configurationChanged && activeClusterChanged && isGlobalNamespace {
```

Comment:
do we already block local namespace update active cluster?

Reply:
yes, and added a UT to validate